web/src/app/services/job/job.service.ts

```diff
@@ -29,11 +29,32 @@ import {SurveyService} from 'app/services/survey/survey.service';
   providedIn: 'root',
 })
 export class JobService {
+  defaultColors = List([
+    '#F37C22',
```

Comment:
Also, please update the order to reflect the default order in the bug:

Potential order:

1. Orange
2. Blue
3. Yellow
4. Purple
5. Red